test/cli/struct_strict/test.out

```diff
@@ -1,14 +0,0 @@
-test/cli/struct_strict/test.rb:7: The method `foo` does not have a `sig` https://srb.help/7017
-     7 |Bad = Struct.new(:foo) do
-                          ^^^
-  Note:
-    Struct classes defined with `Struct` are hard to use in `# typed: strict` files.
-    Consider using `T::Struct` instead.
-
-test/cli/struct_strict/test.rb:7: The method `foo=` does not have a `sig` https://srb.help/7017
-     7 |Bad = Struct.new(:foo) do
-                          ^^^
-  Note:
-    Struct classes defined with `Struct` are hard to use in `# typed: strict` files.
-    Consider using `T::Struct` instead.
```

Comment:
These errors don't happen anymore—we may want to consider deleting this special handling.